aopy/data/bmi3d.py

```diff
@@ -399,6 +399,35 @@ def load_ecube_digital(path, data_dir):
     metadata = load_ecube_metadata(os.path.join(path, data_dir), 'DigitalPanel')
     return data, metadata
 
+def load_emg_digital(data_dir, emg_data):
+    '''
+    Just a wrapper around load_ecube_data() and load_ecube_metadata()
+
+    Args:
+        data_dir (str): base directory where ecube data is stored
+        emg_data (str): folder you want to load
+
+    Returns:
+        tuple: Tuple containing:
+            | **data (nt):** digital data, arranged as 64-bit numbers representing the 64 channels
+            | **metadata (dict):** metadata (see load_ecube_metadata() for details)
+    '''
+
+    emg_data, emg_metadata = load_bmi3d_hdf_table(data_dir, emg_data, 'data')
+    print(emg_metadata)
+    emg_data_reshape = emg_data.view(('f8', (len(emg_data.dtype),)))
```

Comment:
we were going to check if we can save the data in its original form as int16 instead of converting float64 since it will save a lot of space. we would also need to update this line